### PR TITLE
Added build instructions

### PR DESCRIPTION
**Pull request by isparkes**
Hey, thanks for putting this Docker image together. It helped me out of a spot... ;)

I added a little bit of dox that might help others keep their image up to date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,20 @@ If you see out of date documentation, lack of tests, etc., you can help out by e
...
 Continuous Integration and Continuous Delivery is made possible with the great services from [GitHub](https://github.com), [Travis CI](https://travis-ci.org/), and [CircleCI](https://circleci.com/) written in [Ruby](https://www.ruby-lang.org/), using [RSpec](http://rspec.info/), [Capybara](https://jnicklas.github.io/capybara/), and [PhantomJS](http://phantomjs.org/) frameworks.
+
+## Building the Docker image
+
+JIRA Service Desk is in a period of fast development, and releases are coming thick and fast. In order to keep up to date, you might have to rebuild this image. Here are the steps to do it
+
+Clone the repository:
+```git clone git@github.com:cptactionhank/docker-atlassian-jira-service-desk.git
+cd docker-atlassian-jira-service-desk
+export PATH=$PATH:$(pwd)/bin
```
Comment (cptactionhank) [inline README.md:44]: can we remove this line

File: README.md [hunk 2]
```diff
@@ -33,3 +33,20 @@ If you see out of date documentation, lack of tests, etc., you can help out by e
...
+prepare
```
Comment (cptactionhank) [inline README.md:45]: and change this to `bin/prepare`

**Review by cptactionhank — CHANGES_REQUESTED** [review]
I like your changes but I'm thinking perhaps moving it to a separate file since the README is used as content body on Docker Hub and to keep it and sweet.

How about `HOWTO.md` or `BUILD.md`

**Comment by isparkes** [comment]
You da man.

I mostly wanted to say thanks for putting it together, it saved a stack of
time, and it was such a great job, the only little improvement was this.
Either HOWTO or BUILD would be great.

Have a good evening, and thanks!

Ian

**Comment by cptactionhank** [comment]
Can you update your pull request to move it to say `HOWTO.md` and commit this as well so I can merge your pull request and you get credit in the commit history?

**Comment by isparkes** [comment]
Yep, will do this morning!

Ian

**Pull request closed without merging**